Resources/public/mapbender.element.digitizer.js

```diff
@@ -3353,8 +3353,13 @@
             var attributes = feature.attributes;
             var tableName = schema.featureType.table;
             var relativeWebPath = Mapbender.configuration.application.urls.asset;
-            window.open(relativeWebPath + widget.options.fileUri + '/' + tableName + '/' + attributeName + '/' + attributes[attributeName]);
 
+            if (schema && schema.featureType && schema.featureType.files[0] && schema.featureType.files[0].hasOwnProperty('uri')) {
```

Comment:
Why schema.featureType.files[0]? Why is this even an array if we only acces the frist index?